src/Relay/Connection/Paginator.php

```diff
@@ -127,11 +130,11 @@ private function handleEntities($entities, callable $callback)
      */
     private function computeTotalCount($total, array $callableArgs = [])
     {
-        if (null !== $this->totalCount) {
```

Comment:
why this change? both are not equivalent

Reply:
In PHP 7.4 it's the same. The property `totalCount` is declared as `private int $totalCount` and can never be null. 
If you try to test like this `null !== $this->totalCount` when the property is not set you will get the error:

```
Uncaught Error: Typed property Test::$total must not be accessed before initialization
```
so it only can either be int or unset.